backend/apps/owasp/services/certificate_service.py

```
@@ -0,0 +1,61 @@
+"""Certificate service layer for Contributor Recognition Program."""
```

Ruff (0.15.15)

[warning] File `backend/apps/owasp/services/certificate_service.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)